ios/Podfile

```diff
@@ -60,7 +60,9 @@ def bt_pods
   permissions_path = '../node_modules/react-native-permissions/ios'
   pod 'Alamofire', '~> 4.9.1'
   pod 'ZIPFoundation'
+  pod 'Firebase/Analytics'
```

Comment:
I'm not sure if we are supposed to include this package for security concerns, @mattThousand thoughts?

Reply:
Crashlytics for iOS was set up in #11 actually 😄  - this scope of this PR should be for Android